generals/agents/agent_factory.py

```diff
@@ -0,0 +1,25 @@
+from .random_agent import RandomAgent
+from .expander_agent import ExpanderAgent
+from .agent import Agent, EmptyAgent
+
+class AgentFactory:
+    """
+    Factory class for creating agents.
+    """
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def make_agent(agent_type: str, **kwargs) -> Agent:
```

Comment:
It might be nice to name it simply `make`. We know that it will make an `agent` since it is `AgentFactory`.

Reply:
true, will do in later updates, thanks